src/main/java/org/influxdb/impl/InfluxDBImpl.java

```diff
@@ -212,6 +230,43 @@ public void write(final String database, final String retentionPolicy, final Con
    * {@inheritDoc}
    */
   @Override
+  public void write(final int udpPort, final String records) {
+    initialDatagramSocket();
+    byte[] bytes = records.getBytes(Charsets.UTF_8);
```

Comment:
@majst01  Done with readme change and javadoc improve (for this method, it use {@inheritDoc} . the javadoc is located  in interface method)
 Thanks.